src/test/scala/ai/starlake/utils/ParserSQLTest.scala

```diff
@@ -0,0 +1,50 @@
+package ai.starlake.utils
+
+import ai.starlake.TestHelper
+import org.apache.spark.sql.catalyst.parser.ParserSQL
```

Comment:
Excellent 
As I understand the Spark parser is able to handle BQ syntax ?

Reply:
most of it yes, except for some keywords which were identified and managed in SQL Parser